classes/event/instance.php

```diff
@@ -209,7 +203,7 @@ protected function _format_return(array $calls, $return_type)
 				return json_encode($calls);
 				break;
 			case 'none':
-				return;
+				return false;
```

Comment:
@stevewest @WanWizard 
Sorry, I was misunderstanding design and specification of PHP. I fixed it, please peer-review again.